app.js

```diff
@@ -0,0 +1,18 @@
+const express = require('express');
+const app = express();
+
+const { getTopics } = require("./controllers/topics-controllers.js");
+
+app.get('/api/topics', getTopics);
+
+//Unavailable Route (404) Error Handling
+app.all('*', (req, res) => {
+    res.status(404).send({msg: "Invalid Endpoint"});
```

Comment:
Nice implementing and testing of a custom message for non-existent endpoints 🙌

Reply:
Thank you!